content/06.know-your-problem.md

```diff
@@ -19,13 +19,16 @@ Study designs will often have different assumptions and caveats, and these canno
 Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated.
 Samples originating from the same study site, oversampling of ethnic groups or zip codes, and sample processing differences are all sources of variation that need to be accounted for.
 
+In all cases, investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by thoroughly inspecting the input data.
+Data exploration with unsupervised learning and data visualization can reveal the biases and technical artifacts in these datasets, providing a critical first step to assessing data quality before any deep learning model is applied.
+In some cases, these analyses can identify biases from known technical artifacts or sample processing which can be corrected through preprocessing techniques to support more accurate application of deep leaning models for subsequent prediction or feature identification problems from those datasets.

```

Comment:
I would remove "from those datasets" (I think it's implied)
"deep learning" -> "DL"

Reply:
My 2 cents: I would go the other direction - i think we should just use "deep learning" as the written out text throughout.

Reply:
Either way is fine! No strong opinion from me as long as we're being consistent.

Reply:
I also prefer writing out "deep learning".  For a short paper, we can avoid most or all acronyms.

Reply:
We're already well over the editor's suggested word limit of 2500-3000 words. I personally prefer to eliminate the use of "DL" for "deep learning" for several reasons in order of strength:

1. Fewer acronyms to keep in mind for the reader
2. DL isn't a super common acronym in general. For example, when you google "ML" the first thing you get is Merrill Lynch and the next thing is a bunch of videos on TF followed by an ML cause from Stanford. For "DL", nothing deep learning even shows up.
3. I'm not entirely sure if this applies to academic papers, but search engine optimization is worth considering. For search methods which rely on term frequency of body content, using the keyword "deep learning" more will make the paper more visible than only having one or two occurrences.

Since there are at least three authors (@cgreene, @agitter, and myself) in favor of the change and none opposed, I think we should do it. I've made a new issue (#264) to track it. In the meantime, we can be inconsistent until normalization after we freeze the content (once these PRs are done).